From 3eca0020d80cebbe9a344717781fc92151068bd4 Mon Sep 17 00:00:00 2001
From: Lukas Rusak <lorusak@gmail.com>
Date: Sat, 25 Jan 2020 07:42:27 -0800
Subject: [PATCH] configure.ac: remove weak_import check for osx

---
 configure.ac | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/configure.ac b/configure.ac
index db1ce841f..c250df819 100644
--- a/configure.ac
+++ b/configure.ac
@@ -122,14 +122,6 @@ case "$host" in
   ;;
   *darwin*)
     have_macosx=yes
-    save_LDFLAGS="$LDFLAGS"
-    dnl Try to use -no_weak_imports if available. This makes sure we
-    dnl error out when linking to a function that doesn't exist in the
-    dnl intended minimum runtime version.
-    LDFLAGS="$LDFLAGS -Wl,-no_weak_imports"
-    AC_MSG_CHECKING([whether the linker supports -Wl,-no_weak_imports])
-    AC_LINK_IFELSE([AC_LANG_PROGRAM([], [])],
-      [AC_MSG_RESULT(yes)], [AC_MSG_RESULT(no); LDFLAGS="$save_LDFLAGS"])
   ;;
   *solaris*)
     have_elf=yes
-- 
2.24.1

